### DISCUSSION TITLE
Find out how many new words we added to our website translation last year

### DISCUSSION THREAD

**Discussion by emmapeel2** [WeblateOrg/weblate#10962] [Q&A] [answered]
I would like to easily establish how many new words we added to our website translations last year, so I can have an idea of how many updates we do.

I have been able to generate a search, and I can see we added 238 strings to this component: https://hosted.weblate.org/translate/tor/support-portal/en/?q=language%3Aen+added%3A%3E%3D2023-01-08&sort_by=-timestamp&offset=238

But I cannot find how many **words** that was.

I would like to have the possibilities of generating reports about source strings, maybe in 'insights/translation reports'.

**Comment by nijel** [comment, marked as answer]
This information is tracked by Weblate, there is just no way to get this out. I've created https://github.com/WeblateOrg/weblate/issues/10964 to track this feature request.
